src/etaoin/api.clj

```diff
@@ -2240,6 +2239,11 @@
   `(when-not (~predicate)
      ~@body))
 
+(defmacro when-not-several-drivers
```

Comment:
It could be just `when-not-drivers` because it already carries "s" at the end of it (whereas `when-not-driver` means one driver).